backend/src/main/java/com/webappsbusters/parcelmanagement/controller/ParcelController.java

```diff
@@ -0,0 +1,32 @@
+package com.webappsbusters.parcelmanagement.controller;
+
+import com.webappsbusters.parcelmanagement.domain.ParcelDto;
+import com.webappsbusters.parcelmanagement.mapper.ParcelMapper;
+import com.webappsbusters.parcelmanagement.service.ParcelService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.server.ResponseStatusException;
+
+@RestController
+@RequestMapping("/v1")
+public class ParcelController {
```

Comment:
Please add `@CrossOrigin(origins = "*", allowedHeaders = "*")`, withoout we won't be able to access it from frontend

Reply:
Done